cpp/board.h

```diff
@@ -185,6 +189,30 @@ namespace gb{
         return false;
       }
 
+      bool Game_Over(){
+        for(int row = 0;row<4;row++){
+          for(int col = 0;col<4;col++){
+                 if(tiles[row][col] == 0) return false;
```

Comment:
change indent width to `2` to match the rest.

Also for the following `if` conditions.